src/AutoRest.CSharp/Common/Input/CadlInputEnumTypeConverter.cs

```diff
@@ -50,7 +50,13 @@ public static InputEnumType CreateEnumType(ref Utf8JsonReader reader, string? id
             }
 
             name = name ?? throw new JsonException("Enum must have name");
-            description = description ?? throw new JsonException("Enum must have a description");
+            // TODO: roll back to throw JSON error when there is linter on the upstream to check enum without @doc
+            //description = description ?? throw new JsonException("Enum must have a description");
+            if (description == null)
+            {
+                description = "";
+                System.Console.Error.WriteLine($"[Warn]: Enum '{name}' must have a description");
```

Comment:
Shouldn't we write it into `Out` rather than `Error`? It doesn't break the execution and there is nothing we can do now with this error.

Reply:
@AlexanderSher It's a best practice to send information to `StdErr`. See: https://clig.dev/ Sending message to `StdErr` won't break the execution, the returning value will.